spacy/training/loggers.py

```diff
@@ -45,8 +70,9 @@ def setup_printer(
             cols=["E", "#"] + loss_cols + score_cols + ["Score"],
             widths=[3, 6] + [8 for _ in loss_cols] + [6 for _ in score_cols] + [6],
         )
-        write(msg.row(table_header, widths=table_widths, spacing=spacing))
-        write(msg.row(["-" * width for width in table_widths], spacing=spacing))
+        if print_progress:
```

Comment:
is there more code we can have as part of the new `if` here? Like we don't need to format the table if we'll never use it.